docs/source/development.rst

```diff
@@ -73,23 +73,10 @@ the `ANTLR repository release 4.9.2
 and the rest of the files in ``pytsql/src/pytsql/grammar`` are generated by ``antlr`` or
 ``speedy-antlr-tool``.
 
-You can generate these files by running the following commands in
+You can generate these files by running the following steps in
 ``pytsql/src/pytsql/grammar`` directory after adapting the path to your
 respective ``antlr`` jar file as follows:
 
-::
-
-    java -jar /usr/local/lib/antlr-4.9.2-complete.jar -Dlanguage=Cpp -visitor -no-listener -o cpp_src tsql.g4
-    java -jar /usr/local/lib/antlr-4.9.2-complete.jar -Dlanguage=Python3 -no-visitor -no-listener -o . tsql.g4
-
-And then running the ``speedy-antlr-tool`` in the same directory using Python
-
-::
-
-    from speedy_antlr_tool import generate
-
-    generate(
-        py_parser_path="tsqlParser.py",
-        cpp_output_dir="cpp_src",
-        entry_rule_names=["tsql_file"],
-    )
+1. Adapt the path variable `$antlr4` to your respective antlr jar file in `helper_generate_parser.sh`
+2. Run `bash helper_generate_parser.sh`
+3. (Run `pip install --no-build-isolation -e .` in the project root afterwards)
```

Comment:
Why the `()` here?

Reply:
I would also install it without the `-e` flag because the C++ code will not be editable anyways. I assume the brackets show that this is not necessary, and it is just to test the installation, but I would state that explicitly.